dotnet/src/SemanticKernel.Abstractions/SemanticKernel.Abstractions.csproj

```diff
@@ -30,4 +30,8 @@
     <InternalsVisibleTo Include="SemanticKernel.Extensions.UnitTests" />
     <InternalsVisibleTo Include="DynamicProxyGenAssembly2" /> <!-- Moq -->
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Prompts\" />
+  </ItemGroup>
```

Comment:
Why do we need this?

Reply:
We don't sneaked in from merges. Thanks for spotting.